RELEASE.md

```diff
@@ -51,5 +51,12 @@ The Appsody CLI is made available by creating a tagged GitHub release
 1. Go to the [appsody/website](https://github.com/appsody/website/branches) repo and create a PR for the new Travis build branch.
 1. Review and merge the PR.
 
+## Inform users about the new release
```

Comment:
How about:

"Post a message to the `cli` slack channel about the new release, providing a link to the relevant release notes"